sway-core/src/language/ty/declaration/declaration.rs

```diff
@@ -378,6 +378,17 @@ impl TyDecl {
         }
     }
 
+    pub fn get_trait_decl_ref(&self) -> Option<DeclRefTrait> {
```

Comment:
Nitpick, the implementation sticks out and breaks the pattern that all other `get_xyz_decl_ref` methods have :wink: